Formula/zsh-syntax-highlighting.rb

```diff
@@ -17,23 +17,6 @@ def install
     system "make", "install", "PREFIX=#{prefix}"
   end
 
-  def caveats
-    <<-EOS.undent
-    To activate the syntax highlighting, add the following at the end of your .zshrc:
-
-      source #{HOMEBREW_PREFIX}/share/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh
```

Comment:
Let's show just the first line of text and the first `source` and skip the rest.